contracts/upkeeps/CronUpkeep.sol

```diff
@@ -168,7 +159,13 @@ contract CronUpkeep is
    * @return list of active cron job IDs
    */
   function getActiveCronJobIDs() external view returns (uint256[] memory) {
-    return s_activeCronJobIDs;
+    uint256 length = EnumerableSet.length(s_activeCronJobIDs);
+    uint256[] memory jobIDs = new uint256[](length);
+    uint256 idx;
+    for (idx = 0; idx < length; idx++) {
```

Comment:
nitpick - I think you can make this''' `for (uint256 idx = 0; ...`

Reply:
Done